src/nvim/message.c

```diff
@@ -3114,14 +3114,14 @@ int vim_vsnprintf(char *str, size_t str_m, char *fmt, va_list ap, typval_T *tvs)
 {
   size_t str_l = 0;
   bool str_avail = str_l < str_m;
-  char        *p = fmt;
+  char *p = fmt;
   int arg_idx = 1;
 
-  if (p == NULL)
+  if (!p)
```

Comment:
@elmart Yes, we settled on that long ago. https://github.com/neovim/neovim.github.io/issues/53